### DISCUSSION TITLE
Use nangste45 platform to run C906 design

### DISCUSSION THREAD

**Discussion by qiubei0522** [The-OpenROAD-Project/OpenROAD-flow-scripts#1728] [Q&A]
How to solve the following errors when using nangste45 platform to run C906 design?
![eea9b01511b5740b8b4ca9917106923](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/75148042/8039edc3-c174-4a76-bf31-09d6800a7ca4)

**Comment by maliberty** [comment]
You should start by looking at the congestion in the GUI as the message suggests.  Also package a test case with 'make global_route_issue'

**Reply by qiubei0522** [reply]
I understand what you mean. I will upload the data after packing it

**Reply by qiubei0522** [reply]
Since the data content is too large, how should I send it to you

**Reply by maliberty** [reply]
Put it in google drive and post a link here.
